modules/boss/src/main/ts/ephox/boss/mutant/Properties.ts

```diff
@@ -41,6 +41,10 @@ const isSpecial = (item: Gene): boolean => {
   return item.name === GeneTypes.Special;
 };
 
+const getLanguage = (item: Gene): Optional<string> => {
+  return Obj.get(item.attrs, 'lang');
+};
```

Comment:
```suggestion
const getLanguage = (item: Gene): Optional<string> =>
  Obj.get(item.attrs, 'lang');
```
Just giving an actual suggestion for Diogo's comment, as he's right that we do prefer shorthand (though it should be over 2 lines).